### ISSUE TITLE
Keyboard shortcuts

### ISSUE THREAD

**Issue by chirimoya** [sulu-io/sulu#1093]
Some global keyboard shortcuts would make working with sulu more efficient:

ctrl+f: search
ctrl+s: save

**Comment by danrot** [comment]
@chirimoya What about this issue? Do we still want shortcuts like this? For me it always feels weird to "hide" browser functionality like that...

**Comment by normenrollf** [comment]
Yes, I want shortcuts like this … and i know other people who would like that.

**Comment by julia-vw** [comment]
I cannot find anything about keyboard shortcuts in Sulu cms. Are there any? Is there way to create/add them?

**Comment by alexander-schranz** [comment]
@julia-vw Currently such shortcuts as listed here in the issue are not implemented. Any feedback here is welcome what type of shortcuts you maybe want in Sulu.

Inside the `Texteditor` there are shortcuts provided by CKEditor, you see the list of shortcuts here for it: https://ckeditor.com/docs/ckeditor5/latest/features/keyboard-support.html

**Comment by julia-vw** [comment]
I'd like to have a shortcut for 'Save an Publish' at least. We also have a lot of nested elements, so it would be nice to be able to  'Collapse all'.
